lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -333,6 +354,7 @@ class LighthouseReportViewer {
    * @private
    */
   _loadFromGistURL(urlStr) {
+    this._toggleLoadingBlur(true);
```

Comment:
this call shouldn't be needed, right? Since it calls `this._loadFromDeepLink()`

Reply:
fo sho